### DISCUSSION TITLE
Having errors using evaluate from ExpandingWIndowSplitter

### DISCUSSION THREAD

**Discussion by ckwan48** [alan-turing-institute/sktime#3182] [Q&A]
I am trying to validate my model using ExpandingWIndowSplitter, but I keep getting an 'ValueError: Must supply freq for datetime value' error message when it runs evaluate. I'm not sure what the issue is? I am using quarterly data to test it.

```
forecaster = STLForecaster(seasonal=5)
        
cv = ExpandingWindowSplitter(step_length=1, fh=list(range(1,5)), initial_window=1)
df = evaluate(forecaster=forecaster, y=train.squeeze(), cv=cv, strategy="refit")
```

**Comment by khrapovs** [comment]
@ckwan48 Could you, please, supply a complete runnable code snippet even if it returns the `ValueError`?

**Reply by fkiraly** [reply]
@ckwan48, what @khrapovs means, it should include all imports, and code that produces the `train` data. We will then use that in debugging.
